media/js/base/stub-attribution.js

```diff
@@ -369,28 +420,33 @@ if (typeof window.Mozilla === 'undefined') {
     };
 
     /**
-     * A crude check to see if Google Analytics has loaded. Periodically
-     * attempts to retrieve the client ID from the global `ga` object.
+     * A crude check to see if Google Analytics has loaded.
      * @param {Function} callback
      */
-    StubAttribution.waitForGoogleAnalytics = function (callback) {
```

Comment:
If you're renaming things then you might need to check this does not break the RTAMO code at https://github.com/mozilla/bedrock/blob/main/media/js/firefox/new/common/thanks-direct.js